src/DistributedComputations/distributed_fft_based_poisson_solver.jl

```diff
@@ -55,155 +47,159 @@ Algorithm for two-dimensional decompositions
 For two-dimensional decompositions (useful for three-dimensional problems),
 there are three forward transforms, three backward transforms,
 and four transpositions requiring MPI communication. In the schematic below, the first
-dimension is always the local dimension. In our implementation of the PencilFFTs algorithm,
-we require _either_ `Nz >= Rx`, _or_ `Nz >= Ry`, where `Nz` is the number of vertical cells,
+dimension is always the local dimension. In our implementation we require
+`Nz ≥ Ry` and `Nx ≥ Ry` with the additional constraint that `Nz % Ry = 0` and `Ny % Rx = 0`.
```

Comment:
Same comment as before. Should this be
```suggestion
`Nz ≥ Ry` and `global_grid.Nx ≥ Ry` with the additional constraint that `Nz % Ry = 0` and `global_grid.Ny % Rx = 0`.
```
?

Reply:
Yes, I have defined `Nx`, `Ny`, `Rx` and `Ry` at the beginning of the docstring